tests/foreman/cli/test_activationkey.py

```diff
@@ -619,17 +620,19 @@ def test_positive_usage_limit(module_org, module_location, target_sat):
             'lifecycle-environment-id': env['id'],
             'content-view': new_cv['name'],
             'organization-id': module_org.id,
-            'max-hosts': '1',
+            'max-hosts': max_hosts,
         }
     )
-    with Broker(nick='rhel7', host_class=ContentHost, _count=2) as clients:
+    with Broker(nick='rhel8', host_class=ContentHost, _count=2) as clients:
```

Comment:
any reason to flip it on `rhel8` from `rhel7`?

Reply:
`rhel7` becomes a bit obsolete in my eyes so I just wanted to jump 1 version up :)

Reply:
After the latest commit we use the default rhel version now.